client/verta/verta/_deployment/endpoint.py

```diff
@@ -187,3 +188,14 @@ def get_access_token(self):
         if len(tokens) == 0:
             return None
         return tokens[0]['creator_request']['value']
+
+    def get_deployed_model(self):
+        status = self.get_status()
+        if status['status'] != "active":
+            raise RuntimeError("model is not currently deployed (status: {})".format(status))
+
+        access_token = self.get_access_token()
+        return DeployedModel.from_url("{}://{}/api/v1/predict{}".format(
+            self._conn.scheme,
+            self._conn.socket,
+            self.path), access_token)
```

Comment:
Minor suggestion:
Could we do a `url = "..."` in a separate line? Like [this](https://github.com/VertaAI/modeldb/blob/eafe27d/client/verta/verta/_deployment/endpoint.py#L129-L136).
For me, it's a bit easier to read.